src/shared/components/timer/Timer.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
```

Comment:
I think this file is not necessary, I don't see it being used anywhere else

Reply:
Correct, this was just for the old wallet "debug" mode.

Reply:
Deleted 👍